src/openrct2/audio/audio.h

```diff
@@ -78,74 +80,77 @@ struct rct_vehicle_sound_params
     uint16_t priority;
 };
 
-enum RCT2_SOUND
+enum class RCT2Sound : uint8_t
```

Comment:
We can remove RCT2, perhaps just call it `SoundType` or `SoundId` or `SoundKind`?

Reply:
It is sort of tied to RCT2 though. If we ever wanted to add more it would break saves. I would probably call it `SoundId`.

Reply:
Changed to `SoundId`